lib/ui5Framework/npm/Registry.js

```diff
@@ -39,7 +39,13 @@ class Registry {
 		if (!this._npmConfig) {
 			const {default: libnpmconfig} = await import("libnpmconfig");
 			const opts = {
-				cache: this._cacheDir
+				cache: this._cacheDir,
```

Comment:
any test for that?

Reply:
We currently don't have real e2e test. One thing we could at least test is whether the expected options are passed to pacote.

Reply:
That would be fine.